### DISCUSSION TITLE
Diary Type App

### DISCUSSION THREAD

**Discussion by boilermanc** [vercel/community#2517] [Show & Tell]
what would be a good template to start with when wanting to build a diary type app to run on mobile for multiple users?

**Comment by amyegan** [comment]
I don't know of a free template for a multi-user diary app specifically, but there are quite a few Vercel templates that have elements of what you want. You can use them as inspiration and put together the features your app needs. Here are a few I think may be helpful:
- [Platforms Starter Kit](https://vercel.com/templates/next.js/platforms-starter-kit)
- [Subscription Starter](https://vercel.com/templates/next.js/subscription-starter)
- [Auth for subdomains](https://vercel.com/templates/next.js/subdomains-auth)
- [Realtime Chat App](https://vercel.com/templates/next.js/realtime-chat-app)
- [Blog templates](https://vercel.com/templates?type=blog)
